docs/pages/versions/unversioned/sdk/asset.md

```diff
@@ -14,13 +14,59 @@ import PlatformsSection from '~/components/plugins/PlatformsSection';
 
 <InstallSection packageName="expo-asset" />
 
-## API
+# API
 
 ```js
 import { Asset } from 'expo-asset';
 ```
 
-This class represents an asset in your app. It gives metadata about the asset (such as its name and type) and provides facilities to load the asset data.
+## Hooks
+
+### `useAssets`
+
+```ts
+const [assets, error] = useAssets([...]);
+```
+
+load one or many of asset modules to use in your app. After loading these assets, with [`Asset.loadAsync`](#assetloadasyncmodules), you can use these modules to use the downloaded asset.
```

Comment:
Small suggestion, but generally the second sentence seems to be a little bit confusing 😅

Should we call them *modules*? Also it sounds like we still need to call `Asset.loadAsync` to use them 🤔 

```suggestion
Loads one or many of assets to use in your app. After loading these assets, with [`Asset.loadAsync`](#assetloadasyncmodules), you can use these modules to use the downloaded asset.
```

Reply:
Yes, I was doubting about calling them modules as well. Without is definitely better 👍